projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html

```diff
@@ -0,0 +1,78 @@
+<geoengine-sidenav-header>Raster Scaling / Unscaling</geoengine-sidenav-header>
+
+<geoengine-dialog-help>
+    <p>
+        This operator applies scaling or unscaling to a raster data.
+        <br />
+        The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset. This is done by applying the
+        formulas to every pixel.
+        <br />
+        For <b> unscaling </b> the formula is: <i>p_new = p_old * slope + offset</i>. <br />
+        For <b> scaling </b> the formula is: <i>p_new = (p_old - offset) / slope</i>. <br />
+        <i>p_old</i>> and <i>p_new</i> refer to the old and new pixel values. The slope and offset values are either properties attached to
+        the input raster or a fixed value. For more information see
+        <a href="https://docs.geoengine.io/operators/rasterunscaling.html">docs.geoengine.io</a>.
```

Comment:
i think it's rastercaling.hml

Reply:
[71be329](https://github.com/geo-engine/geoengine-ui/pull/471/commits/71be329311aa2f017359ac303d9a7db5baba565b)